app/scripts/components/common/catalog/catalog-content.tsx

```diff
@@ -82,7 +83,7 @@ function CatalogContent({
     prepareDatasets(allDatasetsWithEnhancedLayers, {
     search,
     taxonomies,
-    sortField: DEFAULT_SORT_OPTION,
+    sortField: DEFAULT_SORT_FIELD,
     sortDir: DEFAULT_SORT_OPTION,
     filterLayers: filterLayers ?? false
   }));
```

Comment:
Lets type sortDir in prepareDatasets to be `'asc' | 'desc'` and if you know the options sortField could be too, would be great to type those. I know we could do by name and pubDate